index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>personal-portfolio</title>
+  </head>
+  <body>
+    <header>
+      <div class="my-logo">
+        <a href="#" class="logo">Rejoice</a>
+        <nav class="menu-bar">
+          <i class="fas fa-bars"></i>
+        </nav>
+      </div>
+    </header>
+
+    <section class="headline-wrap">
+      <div class="flex headline-wrap">
+        <h1 class="greetings">
+          I’m Rejoice <br />Glad to see <br />
+          you!
+        </h1>
+        <p class="headline-profile margin-top-12">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
```

Comment:
- [  Optional ] Great job so far. let's improve this section by ensuring that you follow the font characteristics as presented [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) in the figma design.